push_notification_ui/static/js/notifications.js

```diff
@@ -0,0 +1,16 @@
+import {getAllNotifications} from "./requests.js"
+
+export function getNotificationItems(notification_items) {
+    notification_items.innerHTML = ''
+    getAllNotifications()
+        .then(data => {
+            data.forEach(notification => render(notification, notification_items))
+        });
+}
+
+function render(notification, notification_items) {
+    const option = document.createElement('option')
+    option.id = notification.id
+    option.appendChild(document.createTextNode(`${notification.title}`))
+    notification_items.appendChild(option)
```

Comment:
Nice! Does it handle the case if we click `reload_notification` button more than once? We would not want to fill `option` with duplicated notifications upon refreshing the list.

Reply:
@GeekyShacklebolt  It's working fine, No duplicate list on clicking refresh button multiple times